game-frontend/src/constants/gameObjects.ts

```diff
@@ -0,0 +1,12 @@
+import { UnitType } from '../../../darwin-types/game-objects/Unit';
+import { FoodType } from '../../../darwin-types/game-objects/Food';
+
+const GAME_OBJECT_TYPES: {
+  UNIT: UnitType;
+  FOOD: FoodType;
+} = {
+  UNIT: 'unit',
+  FOOD: 'food',
```

Comment:
Shouldn't we have the same constants in the backend and frontend anyway?

Reply:
Frontend setup does not allow imports from outside src folder :(

Reply:
So just the types can be imported?

Reply:
Okay, makes sense. We just have to keep an eye on it, so we don't have lots of duplicate code.